miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -4608,6 +4612,10 @@ public static void setInteger(@Nonnull Consumer<Integer> setter, Integer value,
     }
   }
 
+  public static void setString(@Nonnull Consumer<String> setter, Integer value) {
+    setter.accept(value == null ? null : value.toString());
+  }
```

Comment:
This shouldn't be necessary. The `BigDecimal` was expressed as a `String` in the DTO in order to avoid floating point precision issues. Since that's not a concern for integers, the DTO field should be changed to `Integer` to match